includes/User.php

```diff
@@ -2648,22 +2648,7 @@ public function getGlobalAttribute( $attribute, $default = null ) {
 	 * @return bool
 	 */
 	private function shouldGetAttributeFromService( $attributeName ) {
```

Comment:
Maybe this function can be tossed altogether in favor of directly calling `$this->isPublicAttribute( $attributeName );`?